pac-aurora-provisioner/README.md

```diff
@@ -1,56 +1,63 @@
-# PAC Aurora Provisioning - manual steps
+# PAC Aurora Provisioning
 
 The provisioning process will:
 
-* Creates a cluster in eu-west-1 using the cloudformation template
-* Creates a read replica in us-east-1 using the aws cli. This will be updated with a cloudformation template
+* Create a DB parameter group and DB cluster parameter group in eu-west-1 and us-east-1.
+* Create DB subnet group in us-east-1.
+* Creates a cluster in eu-west-1 using CloudFormation. This is formed of two DB instances in two local AZs.
+* Creates a read replica cluster in us-east-1. This is also formed of two DB instances in two local AZs.
+* Creates DNS entries via Konstructor for both regional clusters.
 
-The decomissioning process will:
+The decommissioning process will:
 
-* Delete the Cluster in eu-west-1
-* Deletes the Read Replica in us-east-1
-* Deletes the db parameter and cluster parameter groups
+* Delete the DB parameter groups and cluster parameter groups in both regions.
+* Delete the cluster in eu-west-1
+* Deletes the Read Replica cluster in us-east-1
+* Deletes the DNS entries via Konstructor in both regions.
 
 ## Building the Docker image
 
 The PAC provisioner can be built locally as a Docker image:
 
 `docker build -t pac-provisioner:local .`
 
+## IMPORTANT: Check Configuration
+
+Before proceeding with provisioning or decommissioning an Aurora cluster, please double check the Ansible vault configuration. To do that, first [install Ansible](http://docs.ansible.com/ansible/latest/intro_installation.html) locally, then run the following:
+
+```
+# Environment Type can be 'p', 't' or 'd' depending on the type of cluster you will be working with.
+export ENVIRONMENT_TYPE=p
+ansible-vault edit ./ansible/vaults/vault_${ENVIRONMENT_TYPE}.yml
+```
+
+You will be prompted for a `Vault Password`, this can be found in the **pac-content-provisioner** LastPass note.
```

Comment:
I changed the name of the LastPass note. It is called `pac-aurora-provisioner` now. Had to change it coz keeping a separate a lastpass note for every provisioner would be ideal @peteclark-ft

Reply:
Ah cool - will update this